<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1556</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="formLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1556</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>相机</string>
    </property>
    <addaction name="actionOpenCV"/>
    <addaction name="actionIKapCV"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>图片</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpenImg"/>
    <addaction name="action_SaveImg"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenCV">
   <property name="text">
    <string>OpenCV(USB)</string>
   </property>
  </action>
  <action name="actionIKapCV">
   <property name="text">
    <string>IKapCV</string>
   </property>
  </action>
  <action name="actionOpenImg">
   <property name="text">
    <string>载入图片</string>
   </property>
  </action>
  <action name="action_SaveImg">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
